ingest/annotations.py

```diff
@@ -349,5 +387,8 @@ def validate_numeric_annots(self):
             if annot_type == "numeric" and column_dtype == "object":
                 valid = False
                 msg = f"Numeric annotation, {annot_name}, contains non-numeric data (or unidentified NA values)"
-                self.store_validation_issue("error", "format", msg)
+                self.store_validation_issue(
+                    "error", msg, "content:invalid-type:not-numeric"
```

Comment:
`invalid-foo` is implicit in each level of the issue type value, so using that phrasing here seems inconsistent.

```suggestion
                    "error", msg, "content:type:not-numeric"
```

If that's persuasive, it'd warrant a global find-replace  of `:invalid-type:` with `:type:`.